### DISCUSSION TITLE
How did i make encodarr work without docker ?

### DISCUSSION THREAD

**Discussion by tr1p0p** [BrenekH/encodarr#97] [Q&A] [answered]
Hi !

So i was looking for a tool like yours, and tadarr seem to have issues with arm devices and not your app, and it's seem promising.

But despite all that i don't really understand how can i setup encodarr for, like, my rpi 4 with only the executable. Now i have encodarr controller, i add the path from my video library but after ?

In the wiki it's written "If you are not using containers, you will need mediainfo and FFmpeg in the PATH variable" but how ? What does that mean ? Because like in my debian i have ffmpeg and mediainfo installed and they're in /usr/bin like usual. 

Where i have to put the path variable ? in library ? In the terminal encodarr-controller don't really show a thing.

Thks a lot for the hard work.

**Comment by tr1p0p** [comment]
So, i figure how and where i have to add the variable, in "Path mask" library, with just add "mediainfo, ffmpeg".

At least, for me. Now it's seem everything work fine

**Comment by BrenekH** [comment, marked as answer]
MediaInfo and FFmpeg both need to be accessible to run from anywhere in the system. The way this is done is by having them listed in the PATH environment variable which `apt install mediainfo ffmpeg` does by default (by putting them in `/usr/bin`).

Path masks is different, it allows you to enter a string which Encodarr will use to ignore certain paths. For example, if you have Plex optimize a few movies and don't want Encodarr to change them, adding `Plex Versions` as a path mask will ignore them because Plex places those optimized versions in a folder called `Plex Versions` next to the original file. If you were to add `a` as a path mask, then every path that has an 'a' in it wouldn't be processed (`/myfiles/Movies/Jason Bourne/Jason Bourne.mkv`, `/myfiles/Movies/Batman Begins/Batman Begins.mkv`, and `/myfiles/Movies/Fast & Furious/Fast & Furious.mkv` would all be ignored while `/myfiles/Movies/Chef/Chef.mkv` would be processed).

Also, thanks for checking out Encodarr! It's great to know that it's helpful for more people than just me :smiley:.

**Comment by tr1p0p** [comment]
Ok thanks, so i think i mix up things, now everything run just fine !
